it/src/main/java/org/corfudb/universe/group/CorfuCluster.java

```diff
@@ -0,0 +1,52 @@
+package org.corfudb.universe.group;
+
+import com.google.common.collect.ImmutableList;
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.corfudb.universe.node.Node;
+import org.corfudb.universe.util.ClassUtils;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.corfudb.universe.node.CorfuServer.ServerParams;
+
+public interface CorfuCluster extends Cluster {
+
+    @Builder
+    @EqualsAndHashCode
+    class CorfuClusterParams implements GroupParams {
+        @Getter
+        private String name;
+        @Default
+        private final List<ServerParams> nodes = new ArrayList<>();
+        @Getter
+        private Node.NodeType nodeType;
+        @Default
+        @Getter
+        private final int bootStrapRetries = 3;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)